src/LuaTransformer.ts

```diff
@@ -3200,30 +3200,16 @@ export class LuaTransformer {
         }
     }
 
-    private getAbsoluteImportPath(relativePath: string): string {
-        if (relativePath.charAt(0) !== "." && this.options.baseUrl) {
-            return path.resolve(this.options.baseUrl, relativePath);
-        }
-        return path.resolve(path.dirname(this.currentSourceFile.fileName), relativePath);
-    }
-
     private getImportPath(relativePath: string): string {
-        // Calculate absolute path to import
-        const absolutePathToImport = this.getAbsoluteImportPath(relativePath);
-        if (this.options.rootDir) {
```

Comment:
While I agree that the rootDir Implementation is a bit weird and needs a rework, along side with baseUrl(s)
you should not just remove it.
This will break paths that include files relative from a subdirectory.
For example 2 files `src/lib.ts`  and `src/subdir/subdir2/nested.ts`
nested.ts:
`import *  as lib from "../../lib.ts`

The above should get converted to `require("lib")`. Which I think is what the current implementation does via `path.resolve`.

Probably one of the oldest pieces of code in this project, which could explain why there is no tests for it.

So please also add tests for paths with `..` if you can.

Reply:
Thanks for the info, I'll keep this use case working by adding the code back.

My current tests are just matching the literal string in the require path to what you'd expect in a Lua program.

I don't think I can add add tests for `..` since to test the workaround functionally there may need to be a testing tool to execute a Lua project involving multiple files at different locations.

I'll add the code back but I can't figure out how to build some test cases for it.

Reply:
Figured out a way to test `..` paths, modified `Compiler.ts` to do it.